### PR TITLE
add documentation around the provider param in sales-tax-reporting

### PR DESCRIPTION
**Pull request by ryan-lgtm**
Let's avoid duplicate receipts/STT's/marketplace exempt confusion/provide a quick ref for customers. 😍

### CODE REVIEW

**Comment by ryan-lgtm** [comment]
Quick link - https://deploy-preview-47--taxjar-developers.netlify.app/integrations/sales-tax-reporting/#section-provider

File: source/integrations/sales-tax-reporting.md [hunk 1]
```diff
@@ -48,6 +48,46 @@ Next you’ll need to provide the `transaction_date` parameter. We recommend usi
...
+2. Exempt the order or refund from sales tax, if it includes a supported value.
+
+#### Identify the source of the order or refund
+It is sometimes helpful to track the source of a sale for record purposes. TaxJar accounts have the ability to export transaction data and see the source via the `provider` parameter. For example, `'provider': 'pos'` for on-site sales, or `'provider': 'conference_booth'` for events.
+
+#### Exempting Order or Refund Transactions by Provider
+TaxJar supports identification via `provider` for the following values:
+
+* `'amazon'`
+
```
Comment (cas27) [inline source/integrations/sales-tax-reporting.md:68]: we also support facebook

File: source/integrations/sales-tax-reporting.md [hunk 2]
```diff
@@ -48,6 +48,46 @@ Next you’ll need to provide the `transaction_date` parameter. We recommend usi
...
+* `'ebay'`
+
+* `'etsy'`; or
+
+* `'walmart'`.
+
+These e-commerce channels are marketplaces, and therefore marketplace exempt. The platform handles sales tax calculation and remittance on behalf of the seller.
```
Comment (cas27) [inline source/integrations/sales-tax-reporting.md:75]: This isnt the full story, as it depends on the state and transaction date.  Should we add something about there here?

File: source/integrations/sales-tax-reporting.md [hunk 3]
```diff
@@ -48,6 +48,46 @@ Next you’ll need to provide the `transaction_date` parameter. We recommend usi
...
+
+2. The software supports importing transactions from any of the marketplace platforms above
+
+3. The merchant **is not** connecting a marketplace platform directly to TaxJar, but wants the software to unify sending all transactions to TaxJar
+
+Not taking advantage of the `provider` parameter could result in transaction duplication in TaxJar.
+
+In the event of marketplace sales being made on a platform outside of the above, ensure your software passes an [order-level exemption type](/integrations/sales-tax-reporting/#section-order-level-exemptions) with a value of `'marketplace'`. If one of the supported provider values is provided, it is not necessary to specify an `exemption_type`.
+
+**Note:** It is not necessary to call for sales tax calculations of marketplace orders or refunds. Because they are collected and remitted on behalf of the merchant by the marketplace platform, avoid calling /v2/taxes for known marketplace transactions.
```
Comment (cas27) [inline source/integrations/sales-tax-reporting.md:89]: Again may not be true depending on state and date.

**Review by cas27 — CHANGES_REQUESTED** [review]

File: source/integrations/sales-tax-reporting.md (hunk 1 above)
Comment (ryan-lgtm) [reply source/integrations/sales-tax-reporting.md:68]: @cas27 --

- added facebook
- removed the line regarding marketplace specified provider transactions being marketplace exempt
- replaced said line with an explanation that exemptions are dependent on state/date, with a link to the up-to-date marketing blog content on MF laws
- removed the bottom "note" -- they should hit /v2/taxes so we can determine if sales tax should be collected or not.

Thank you sir!

Comment (fastdivision) [reply source/integrations/sales-tax-reporting.md:68]: @ryan-lgtm We should probably mention Facebook in the API documentation "View Note" tooltips for the `provider` param as well. For this bullet list of supported providers, I'd drop the `; or` and `.` at the end.

Comment (ryan-lgtm) [reply source/integrations/sales-tax-reporting.md:68]: @fastdivision I think this should be good to go now if the explanation sounds good. I also have a PR up on taxjar-api-docs updating the tooltips for orders & refunds and the date in the changelog the feature was supported [here](https://github.com/taxjar/taxjar-api-docs/pull/94).
cc: @cas27

**Review by fastdivision — APPROVED** [review]
Looks good to me 👍

**Review by cas27 — APPROVED** [review]

**Pull request merged**